pulp_container/tests/functional/constants.py

```diff
@@ -70,3 +70,5 @@
 """
 
 DOCKERHUB_PULP_FIXTURE_1 = "pulp/test-fixture-1"
+
+LOCAL_REGISTRY_ADMIN = {'username': 'admin', 'password': 'password'}
```

Comment:
I know it reflects the way it was, but shouldn't this password be retrieved from smash config?